toolchain/check/decl_name_stack.cpp

```diff
@@ -262,20 +262,20 @@ auto DeclNameStack::ApplyAndLookupName(NameContext& name_context,
   }
 
   // For identifier nodes, we need to perform a lookup on the identifier.
-  auto [resolved_inst_id, is_poisoned] = context_->LookupNameInDecl(
+  auto lookup_result = context_->LookupNameInDecl(
       name_context.loc_id, name_id, name_context.parent_scope_id,
       name_context.initial_scope_index);
-  if (is_poisoned) {
+  if (lookup_result.is_poisoned()) {
     name_context.unresolved_name_id = name_id;
     name_context.state = NameContext::State::Poisoned;
-  } else if (!resolved_inst_id.is_valid()) {
+  } else if (!lookup_result.lookup_inst_id().is_valid()) {
```

Comment:
```suggestion
  } else if (!lookup_result.is_found()) {
```

Doing another pass looking at calls

Reply:
Done.